tests/core/mempool/test_mempool.py

```diff
@@ -2220,6 +2220,32 @@ def test_unknown_condition(self, mempool: bool, condition: str, softfork_height:
         else:
             assert npc_result.error is None
 
+    @pytest.mark.parametrize("mempool", [True, False])
+    @pytest.mark.parametrize(
+        "condition, expect_error",
+        [
+            # the softfork condition must include at least 1 argument to
+            # indicate its cost
+            ("(90)", Err.INVALID_CONDITION.value),
+            ("(90 1000000)", None),
```

Comment:
`90` is the `SOFTFORK` condition code. More info here: https://github.com/Chia-Network/chia_rs/pull/168